src/bpdn.jl

```diff
@@ -0,0 +1,20 @@
+function asp_bpdn(A, b, lambda; kwargs...)
+
+"""
+    asp_bpdn(A,B) solves the basis pursuit problem.
+
+    (BP)   minimize_x  ||x||_1  subject to  Ax = B.
```

Comment:
could this again be made a math environment?  (and below as well?)

Reply:
done!